### ISSUE TITLE
Favicon changes after using browser's "Go back"

### ISSUE THREAD

**Issue by nardog** [bluesky-social/social-app#3640]
**Describe the bug**
The favicon for https://bsky.app/ becomes the old one after navigating there through in-tab history.

**To Reproduce**

1. Bookmark https://bsky.app/ on your browser's bookmark bar
2. Open the bookmark
3. Confirm the bookmark's favicon is the blue butterfly
4. Click on any internal link inside the opened tab (e.g. "Search")
5. Use your browser's "Go back"
6. The favicon on the bookmark bar is now the old one (blue shaded rounded rectangle).

**Expected behavior**
The favicon stays the same.

**Screenshots**

**Details**

 - Platform: Desktop Firefox 125
 - App version:

**Additional context**
Also happens with subpages under https://bsky.app/ (e.g. your profile).

**Comment by mary-ext** [comment]
What if you clear your site data?

**Comment by nardog** [comment]
> What if you clear the site data?

Still reproducible. I tried clearing cookies and site data, clearing cache, and deleting and re-adding the bookmark, none of which changed the result.

**Comment by mary-ext** [comment]
I'm still of the idea that it's probably just the browser retaining a stale favicon

**Comment by mozzius** [comment]
Yeah I'm gonna close, sounds like a browser cache issue (they can be extremely aggressive about keeping stale favicons for some reason)
